_includes/releases/v22.2/v22.2.0-alpha.3.md

```diff
@@ -0,0 +1,99 @@
+## v22.2.0-alpha.3
+
+Release Date: September 12, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-0-alpha-3-enterprise-edition-changes">{{site.data.products.enterprise}} edition changes</h3>
+
+- The new `kv.rangefeed.range_stuck_threshold` (default 60s) [cluster setting](../v22.2/cluster-settings.html) instructs RangeFeed clients (used internally by [changefeeds](../v22.2/create-changefeed.html)) to restart automatically if no checkpoint or other event has been received from the server for some time. This is a defense-in-depth mechanism which will log output as follows if triggered: `restarting stuck rangefeed: waiting for r100 (n1,s1):1 [threshold 1m]: rangefeed restarting due to inactivity` [#86820][#86820]
+- Fixed a null pointer exception when [`ALTER BACKUP SCHEDULE`](../v22.2/alter-backup.html) was called after a dependent schedule was dropped. [#87293][#87293]
```

Comment:
Ahhh, `ALTER BACKUP` is a different thing to `ALTER BACKUP SCHEDULE`... there are no docs yet for alter backup schedule yet. If we wanted to link to something, it would be better to link to `create schedule for backup`.

Reply:
Ah ok - added link to 'create schedule for backup'